modin/backends/pandas/query_compiler.py

```diff
@@ -363,6 +363,7 @@ def transpose(self, *args, **kwargs):
     mean = ReductionFunction.register(pandas.DataFrame.mean)
     quantile_for_single_value = ReductionFunction.register(pandas.DataFrame.quantile)
     mad = ReductionFunction.register(pandas.DataFrame.mad)
+    to_datetime = ReductionFunction.register(pandas.to_datetime, axis=1, squeeze=True)
```

Comment:
I thought of a different approach to keep the internal APIs unchanged. What if we made this change and didn't change `ReductionFunction.register`?

```suggestion
    to_datetime = ReductionFunction.register(
        lambda df, *args, **kwargs: pandas.to_datetime(df.squeeze(), *args, **kwargs), 
        axis=1,
    )
```

Reply:
This complicates a little bit the process of function registration. But if we want to keep the internal API's is unchanged as much as possible, it looks very good. I applied the changes.